ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -980,6 +980,43 @@ def test_tensorflow_gather_nd(
     )
 
 
+# tile
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full="True")),
+    dt_and_multiples=helpers.dtype_and_values(
+        available_dtypes=st.sampled_from(["int32", "int64"])
```

Comment:
According to the documentation, `multiples` should be 1-D with length equal to the input's number of dimensions. You should see to those conditions when generating arrays here.

Reply:
noted, I've added some code using ```st.composite``` as you had suggested.

Reply:
more on that thread.